webapp/src/app.tsx

```diff
@@ -1095,7 +1095,11 @@ export class ProjectView
     importHexFile(file: File) {
         if (!file) return;
         pxt.cpp.unpackSourceFromHexFileAsync(file)
-            .done(data => this.importHex(data));
+            .then(data => this.importHex(data))
+            .catch(e => {
+                pxt.log(e);
+                core.warningNotification(lf("Sorry, we could not recognize this file."))
```

Comment:
How does localization work? Is this all I need to add or do I need to add an entry somewhere else as well?
@pelikhan